sample_consensus/include/pcl/sample_consensus/impl/sac_model_line.hpp

```diff
@@ -221,7 +221,12 @@ pcl::SampleConsensusModelLine<PointT>::optimizeModelCoefficients (
 
   // Compute the 3x3 covariance matrix
   Eigen::Vector4f centroid;
-  compute3DCentroid (*input_, inliers, centroid);
+  if (0 == compute3DCentroid (*input_, inliers, centroid))
+  {
+    PCL_WARN ("[pcl::SampleConsensusModelLine::optimizeModelCoefficients] compute3DCentroid failed (returned 0), possibly because there are no valid inliers.\n");
```

Comment:
Might make sense to reword this to eliminate `possibly`. The only reason for a 0 return is if `inliers.empty()` or `all(points == NaN)`

Reply:
Makes sense, I removed "possibly"